src/GafferImage/Median.cpp

```diff
@@ -152,6 +178,127 @@ Imath::Box2i Median::computeDataWindow( const Gaffer::Context *context, const Im
 	return dataWindow;
 }
 
+void Median::hash( const Gaffer::ValuePlug *output, const Gaffer::Context *context, IECore::MurmurHash &h ) const
+{
+	if( output == pixelOffsetsPlug() )
+	{
+		const V2i radius = radiusPlug()->getValue();
+		const V2i tileOrigin = context->get<V2i>( ImagePlug::tileOriginContextName );
+		const Box2i tileBound( tileOrigin, tileOrigin + V2i( ImagePlug::tileSize() ) );
+		const Box2i inputBound( tileBound.min - radius, tileBound.max + radius );
+
+		Sampler sampler(
+			inPlug(),
+			// This plug should only be evaluated with channel name already set to the driver channel
+			context->get<std::string>( ImagePlug::channelNameContextName ),
+			inputBound,
+			(Sampler::BoundingMode)boundingModePlug()->getValue()
+		);
+		sampler.hash( h );
+		h.append( radius );
+		h.append( tileOrigin );
+	}
+	ImageProcessor::hash( output, context, h );
```

Comment:
This should come first, so all hashes are first initialised with the node type and plug name, and only then are the other inputs hashed in.